src/robusta/core/model/base_params.py

```diff
@@ -110,6 +111,30 @@ class AIInvestigateParams(HolmesParams):
     context: Optional[Dict[str, Any]]
 
 
+class HolmesWorkloadHealthParams(HolmesParams):
+    """
+    :var ask: Override question to ask holmes
+    :var resource: The resource related to this investigation. A resource has a `name` and `kind`, and may have `namespace` and `node`
+    :var alert_history: fetch historical alert data on the resource
+    :var alert_history_since_hours: Timespan of historic data to use in hours. 24 by default.
+    :var stored_instrucitons: Use remote instructions specified for the workload.
+    :var instructions: List of extra instructions to supply.
+
+
+    :example ask: What are all the issues in my cluster right now?
+    """
+
+    ask: Optional[str]
+    resource: Optional[ResourceInfo] = ResourceInfo()
+    alert_history: bool = True
+    alert_history_since_hours: float = 24
+    stored_instrucitons: bool = True
+    instructions: List[str] = []
+    include_tool_calls: bool = True
+    include_tool_call_results: bool = True
+    prompt_template: str = "builtin://generic_ask.jinja2"
```

Comment:
Why do we need the prompt override option?
If we do need it, maybe it's better to have it None, and take the default from Holmes?

Reply:
ill remove it not sure its that necessary